app/assets/stylesheets/custom.scss

```diff
@@ -256,3 +252,50 @@ input, select, .uneditable-input {
     display: none;
   }
 }
+
+#myModalBooking {
+  header{
+    display: none; 
+  }
+  .modal-title{
+    text-align: center;
+    color: $blue;
+  }
+  .modal-dialog {
+    width: 40%;
+  }
+  .form-control {
+    background-color: #fff;
+    box-shadow: none;
+    border-radius: 40px;
+    width: 100px;
+    margin-top: 10px;
+  }
+  
+  td{
+    font-family: sans-serif;
+    padding: 10px;
+    font-weight: 600;
+    color: #354052;
+    font-size: 14px;
+    margin: 5px;
+  }
+  #order_rental_period{
+    display: inline;
+  }
+}
+
+h1#show_name{
+  text-align: left;
+  color: $red;
+}
+
+#show_order.col-sm-3{
+  a{
+    text-align: center;
+  }
+}
+#user_profile_show.col-md-12 .col-sm-7{
```

Comment:
file css này e format lại cho chuẩn đi e. trc { có cách hay k? thống nhất toàn file. dưới mỗi block css thì có cách dòng k? để cho đồng bộ nhé

Reply:
Khuyến nghị thì nên có space trước {